scripts/add-cluster.sh

```diff
@@ -309,7 +309,7 @@ spec:
 "
 
 echo "Creating ToolchainCluster representation of ${JOINING_CLUSTER_TYPE} in ${CLUSTER_JOIN_TO}:"
-echo ${TOOLCHAINCLUSTER_CRD}
+echo ${TOOLCHAINCLUSTER_CRD} | yq '.spec.caBundle="***"'
 
```

Comment:
I think that we don't need to print the content of the CR at all - I don't remember that I would need this information in the last years
```suggestion
```

Reply:
ok, so that's even simpler, then